integration/test/src/org/broadinstitute/reaper/server/CromwellContainer.scala

```diff
@@ -0,0 +1,26 @@
+package org.broadinstitute.reaper.server
+
+import com.dimafeng.testcontainers.GenericContainer
+import org.testcontainers.containers.wait.strategy.Wait
+import org.testcontainers.images.builder.ImageFromDockerfile
+
+object CromwellContainer {
+
+  val container = new GenericContainer(
+    new ImageFromDockerfile("cromwell_1")
+      .withDockerfileFromBuilder(builder => {
+        builder
+          .from("openjdk:8-slim")
+          .env("CROMWELL_VERSION", "34")
```

Comment:
Should this also be a test input?

Is this done using an env variable so it can be overridden?

Reply:
can use an env var or conf file either way it shouldn't be hard coded.